web/src/test/java/com/graphhopper/http/GraphHopperServletIT.java

```diff
@@ -138,6 +140,23 @@ public void testGraphHopperWeb() throws Exception {
     }
 
     @Test
+    public void testNonTranslatedInstructions() {
+        GraphHopperAPI hopper = new GraphHopperWeb();
+        assertTrue(hopper.load(getTestRouteAPIUrl()));
+        GHResponse rsp = hopper.route(new GHRequest(42.554851, 1.536198, 42.510071, 1.548128));
+        assertEquals("Continue onto Carrer Antoni Fiter i Rossell", rsp.getBest().getInstructions().get(2).getName());
+
+        GHRequest request = new GHRequest(42.554851, 1.536198, 42.510071, 1.548128);
+        request.getHints().put("translate_instructions", false);
+        rsp = hopper.route(request);
+        assertEquals("Carrer Antoni Fiter i Rossell", rsp.getBest().getInstructions().get(2).getName());
+        // TODO I think we should remove this later on?
+        TranslationMap tr = new TranslationMap().doImport();
+        // TODO First character is small, this is different from web... Maybe we should return the first char as upper case with getTurnDescription?
+        assertEquals("continue onto Carrer Antoni Fiter i Rossell", rsp.getBest().getInstructions().get(2).getTurnDescription(tr.getWithFallBack(Locale.UK)));
```

Comment:
Mhm, I think this PR is about making GraphHopper and GraphHopperWeb instructions behave equally. It is not really about showing street names, even though this is the title of the PR :). When fetching non translated instructions we can return the street name. WDYT does this makes sense?